minio_storage/storage.py

```diff
@@ -388,82 +389,116 @@ def create_minio_client_from_settings(*, minio_kwargs=None):
 
 @deconstructible
 class MinioMediaStorage(MinioStorage):
-    def __init__(self):
-        client = create_minio_client_from_settings()
-        bucket_name = get_setting("MINIO_STORAGE_MEDIA_BUCKET_NAME")
-        base_url = get_setting("MINIO_STORAGE_MEDIA_URL", None)
-        auto_create_bucket = get_setting(
-            "MINIO_STORAGE_AUTO_CREATE_MEDIA_BUCKET", False
-        )
-        auto_create_policy = get_setting(
-            "MINIO_STORAGE_AUTO_CREATE_MEDIA_POLICY", "GET_ONLY"
-        )
-
-        policy_type = Policy.get
-        if isinstance(auto_create_policy, str):
```

Comment:
This was previously a bug: since even `NONE` passes this test, `auto_create_policy` would always be set to `True`, even though it should be `False` for a `NONE` `MINIO_STORAGE_AUTO_CREATE_MEDIA_POLICY`.

Reply:
Nevermind, I misunderstood this setting.